.github/ISSUE_TEMPLATE/issue_report.yml

```
@@ -22,10 +22,12 @@ body:
   - type: checkboxes
     id: not_an_exploit
     attributes:
-      label: Exploit Reports
-      description: "[If your issue is an ingame exploit that allows players to gain an unfair advantage, or an issue which could affect server stability or security, please file it at this link so that it is safe from prying eyes.](https://www.paradisestation.org/forum/179-exploit-reports/)"
+      label: ⚠️ Could this issue actually an exploit? ⚠️
+      description: "[This is important, if your issue is an ingame exploit that allows players to gain an unfair advantage, or an issue which could affect server stability or security, please file it at this link so that it is safe from prying eyes.](https://www.paradisestation.org/forum/179-exploit-reports/)"
       options:
-        - label: I confirm this issue is not an exploit. (Required)
+        - label: I confirm this issue does not give players a unfair advantage. (Required)
+          required: true
+        - label: I confirn this issue does not affect server stability or security. (required)
```

Consistency my beloved
```suggestion
        - label: I confirm this issue does not give players a unfair advantage. (Required)
          required: true
        - label: I confirn this issue does not affect server stability or security. (Required)
```